tests/models/transformers/test_models_transformer_omnigen.py

```diff
@@ -73,9 +74,9 @@ def prepare_init_args_and_inputs_for_common(self):
             "num_attention_heads": 4,
             "num_key_value_heads": 4,
             "intermediate_size": 32,
-            "num_layers": 1,
+            "num_layers": 20,
```

Comment:
Isn’t it a bit too much for CPUs?

Reply:
Overall model size is still very small (2.6 MB). Adds 2 seconds to the current CPU tests.